themes/docs/src/components/live-editor/autocompletion.ts

```diff
@@ -36,6 +36,18 @@ interface Range {
   endColumn: number
 }
 
+function getPropertySnippet({ name, type }, index = 1) {
+  if (type === 'bool') {
+    return name
+  }
+
+  if (type === 'object') {
+    return `${name}={{\${${index}:}}}`
+  }
+
+  return `${name}="\${${index}:}"`
```

Comment:
I would make this string specific and default to `{\${1:}}` (`name={<cusor>}`) as this is the most generic form to pass a value to a JSX prop.

Reply:
@zcei but bool attributes not going to have `=`

Reply:
```
function getPropertySnippet({ name, type }, index = 1) {
  let cursor = `="\${${index}:}"`
  
  if (type === 'bool') {
    cursor = ``
  }

  if (type === 'object') {
    return `={{\${${index}:}}}`
  }

  return `${name}${cursor}`
}

```

🤔 🙅

Reply:
Yes, but whaddaboud numbers? Or arrays?

Reply:
Not used within our components yet 🙃 

There are many more prop type we don't properly support. Like `.anyOf()` and others.

Reply:
`anyOf` is just a propsType validation. But if you say none of the above is justed, then ✅